src/publisher/ajson_ingestor.py

```diff
@@ -78,7 +78,7 @@ def _ingest(data, force=False):
         # only update the fragment if this article version has *not* been published *or* if force=True
         update_fragment = not av.published() or force
         fragments.add(av, XML2JSON, data['article'], pos=0, update=update_fragment)
-        fragments.set_article_json(av, quiet=True)
+        fragments.set_article_json(av, quiet=force) # article-json validation occurs here
```

Comment:
> but we want to validate the JSON on ingest for them too

warnings will still be emitted to logs that are sent to loggly and then emailed to us (current situation).